utopia-remix/app/models/projectAccessRequest.server.ts

```diff
@@ -81,17 +82,14 @@ export async function updateAccessRequestStatus(params: {
       },
     })
 
+    // …finally, grant the role.
     if (params.status === AccessRequestStatus.APPROVED) {
-      // get the request
-      const request = await tx.projectAccessRequest.findFirst({
-        where: {
-          project_id: params.projectId,
-          token: params.token,
-        },
-      })
-      ensure(request != null, 'request not found', Status.NOT_FOUND)
-      const userId = request.user_id
-      permissionsService.grantProjectRoleToUser(params.projectId, userId, UserProjectRole.VIEWER)
+      await permissionsService.grantProjectRoleToUser(
+        params.projectId,
+        request.user_id,
+        UserProjectRole.VIEWER,
+      )
     }
+    // NOTE (ruggi): there should be a way to revoke the permission if the request is REJECTED (TODO)
```

Comment:
I did a little fix for the logic in here (https://github.com/concrete-utopia/utopia/pull/5033/commits/aab7c8aa63d25f471313a8db79635d8699e2440d), left this note for the future (or for the present…)